pwnlib/util/nbstreamreader.py

```diff
@@ -0,0 +1,68 @@
+from threading import Thread
+from Queue import Queue, Empty
+
+class NonBlockingStreamReader:
```

Comment:
While I realize you are just playing it safe, I contacted the author since no explicit license was attached to the [gist](https://gist.github.com/EyalAr/7915597#file-nbstreamreader-py) and am awaiting his reply. If there is an issue I'll rewrite the 39 lines of python myself.